src/bot_botimp.c

```diff
@@ -148,27 +148,27 @@ qbool SetAttributesBasedOnSkill (int skill)
 	cvar_fset (FB_CVAR_ACCURACY, 45 - min (skill, 10) * 2.25);
 	cvar_fset (FB_CVAR_DODGEFACTOR, RangeOverSkill(skill, 0.0f, 1.0f));
 	cvar_fset (FB_CVAR_LOOKANYWHERE, RangeOverSkill(skill, 0.0f, 1.0f));
-	cvar_fset (FB_CVAR_LOOKAHEADTIME, RangeOverSkill(skill, 5.0f, 30.0f));
-	cvar_fset (FB_CVAR_PREDICTIONERROR, RangeOverSkill(skill, 1.0f, 0.0f));
-	cvar_fset (FB_CVAR_DISTANCEERROR, RangeOverSkill(skill, 0.15f, 0.0f));
+	cvar_fset (FB_CVAR_LOOKAHEADTIME, RangeOverSkill(skill, 0.5f, 30.0f));
```

Comment:
Is this on purpose a division of the original value? 0.5f instead of 5.0f
Every other change is a multiplication of the original value.

Reply:
It was on *purpose* - but it may not be *correct* (i.e. I may misunderstand the effect of what I was doing)

My thinking was while most ranges are from (some-large-x, 0) a few are (some-large-x, some-small-y) and THIS one was (some-small-x, some-large-y) - so if I want the bottom at skill zero to be able to get "worse" I figured It should go *smaller*

Since everything was going for "an order of magnitude" a figured a "divide by 10" was the way to go in this case

What do you think?